src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -196,26 +196,34 @@ export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
         case cartError instanceof PaymentQRDeformedError ||
           cartError instanceof PaymentQRMissingInfoError ||
           cartError instanceof PaymentQRUnsupportedError:
-          showErrorAlert(cartError, () => clearCartError());
+          showErrorAlert(new Error(cartError.message), () => clearCartError());
```

Comment:
why do we need to wrap the message with a new error here?

Reply:
The error wrapper is not needed in this case and have removed it. Thanks!